framework/base/Widget.php

```diff
@@ -104,7 +104,13 @@ public static function end()
     {
         if (!empty(self::$stack)) {
             $widget = array_pop(self::$stack);
-            if (get_class($widget) === get_called_class()) {
+
+            $calledClass = get_called_class();
+            if(Yii::$container->has($calledClass) && isset(Yii::$container->getDefinitions()[$calledClass]['class'])) {
```

Comment:
missed white space: `if (`

Reply:
thanks for spotting, added the white space